OpenTaiko/src/Common/CConfigIni.cs

```diff
@@ -1871,21 +1750,25 @@ public CConfigIni(string iniファイル名)
 		// メソッド
 
 		public void RemoveDuplicateKeyAssignments(EInputDevice DeviceType, int nID, int nCode, EKeyConfigPad pad) {
-			var isMenu = pad == EKeyConfigPad.Decide || pad == EKeyConfigPad.RightChange || pad == EKeyConfigPad.LeftChange;
+			var isMenu = pad == EKeyConfigPad.Decide || pad == EKeyConfigPad.RightChange ||
+			             pad == EKeyConfigPad.LeftChange;

```

Comment:
Should warp per-item because each item contains an un-parenthesized binary operator, or use `switch` expression.

Reply:
Was adjusted in [a16ba70](https://github.com/0auBSQ/OpenTaiko/pull/710/commits/a16ba706eb021a77a1740f82f8bd8aa1f7e9f852). Is that sufficient in your opinion?